octavia_proxy/api/drivers/elbv3/driver.py

```diff
@@ -39,18 +39,24 @@ def get_supported_availability_zone_metadata(self):
         return {"eu-nl-01": "The compute availability zone to use for "
                             "this loadbalancer."}
 
-    def _normalize_lb(self, lb):
```

Comment:
I do not think it makes sense dropping this "normalize" function, cause we are going to have another normalization on flavors and az.

Reply:
ok, revert change